test/DynamoCoreTests/DSEvaluationUnitTestBase.cs

```diff
@@ -121,6 +121,17 @@ protected void AssertNullValues()
             }
         }
 
+        protected void AssertNonNull(string guid)
```

Comment:
Added new test utility method to test if any node is non null. Noticed this will be a useful addition to the existing test framework.